mindsdb/integrations/handlers/linkup_handler/__about__.py

```diff
@@ -0,0 +1,9 @@
+__title__ = 'Linkup'
+__package_name__ = 'linkup'
+__version__ = '1.0.0'
+__description__ = "SDK for integrating Linkup API with Python applications"
+__author__ = 'Linkup'
+__github__ = 'https://github.com/linkup-so/linkup-sdk'
```

Comment:
@ZoranPandovski __github__ and __pypi__ should point to MindsDB's GitHub and PyPI respectively, yes?